diff --git a/src/cppQueue.h b/src/cppQueue.h
index 2012d77..f08bac4 100755
--- a/src/cppQueue.h
+++ b/src/cppQueue.h
@@ -54,6 +54,7 @@ public:
 	**	\param [in] lenQDat - Length of static data queue (in bytes) for static array size check against required size for queue
 	**	\return nothing
 	**/
+	cppQueue() = default;
 	cppQueue(const size_t size_rec, const uint16_t nb_recs=20, const cppQueueType type=FIFO, const bool overwrite=false, void * const pQDat=NULL, const size_t lenQDat=0);
 
 	/*!	\brief cppQueue destructor: release dynamically allocated queue
